chapters/06-smart-contracts-in-ralph.md

```diff
@@ -1467,7 +1469,7 @@ Contract SubDomain(url: ByteVec) {
 }
 ```
 
-In the example above, `Domain` is a contract initialized with a `SubDomain` template ID. The `createSubDomain` method creates a new `SubDomain` sub-contract using the `copyCreateSubContract!` built-in function, with `url` as the sub-contract path. Sub-contracts can also be created with `createSubContract!` and `copyCreateSubContractWithToken!` functions, similar to the `createContract!` and `copyCreateContractWithToken!` built-in functions for regular contracts.
+In the example above, `Domain` is a contract initialized with a `SubDomain` template ID. The `createSubDomain` method creates a new `SubDomain` sub-contract using the `copyCreateSubContract!` built-in function, with `fullUrl` as the sub-contract path. Sub-contracts can also be created with `createSubContract!` and `copyCreateSubContractWithToken!` functions, similar to the `createContract!` and `copyCreateContractWithToken!` built-in functions for regular contracts.
```

Comment:
It should be `fullUrl` instead of `url` since the contract creation line is:

```rust
        let subDomainId = copyCreateSubContract!{callerAddress!() -> ALPH: minimalContractDeposit!()}(
            fullUrl, subDomainTemplateId, immFields, mutFields
        )
```

right?

Reply:
I have updated the example a bit, so it uses the `subDomainUrl` as sub contract path, which is more aligned with the idea that it can be traversed like a sub domain. WDYT?